### PR TITLE
chore: Fixes wrong Evaluator config.

### PR DESCRIPTION
**Pull request by jiyongjung0**
Also changes dataset to the one included in the tfx Python package.

**Description of your changes:**
We moved our example to use Keras APIs, but it wasn't reflected in the example. So I changed the module path and the Evaluator config.

And it seems like `{{kfp-default-bucket}}` is not filled in kfp 1.7, so I changed it to a normal placeholder string to distinguish with machine-filled string.

**Checklist:**
- [ ] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: samples/core/parameterized_tfx_oss/parameterized_tfx_oss.py [hunk 1]
```diff
@@ -24,16 +24,16 @@
...
     ptype=str,
 )
 
 # Path to the CSV data file, under which their should be a data.csv file.
-_data_root = 'gs://ml-pipeline-playground/tfx_taxi_simple/data'
+_data_root = '/opt/conda/lib/python3.7/site-packages/tfx/examples/chicago_taxi_pipeline/data/simple'
 
 # Path of pipeline root, should be a GCS path.
 pipeline_root = os.path.join(
-    'gs://{{kfp-default-bucket}}', 'tfx_taxi_simple', kfp.dsl.RUN_ID_PLACEHOLDER
```
Comment (Bobgy) [inline samples/core/parameterized_tfx_oss/parameterized_tfx_oss.py:36]: This is auto filled in marketplace installation, but not for standalone

Comment (jiyongjung0) [reply samples/core/parameterized_tfx_oss/parameterized_tfx_oss.py]: Oh! That was the difference. I'll rollback to '{{kfp-default-bucket}}'. Thanks!

**Comment by Bobgy** [comment]
/lgtm
/approve
Thank you for the fix!

**Pull request merged**